### ISSUE TITLE
Parsing .aag files failed

### ISSUE THREAD

**Issue by Acdimy** [mvcisback/py-aiger#124]
Hi,

I tried the demo program given in py-aiger-abc:

```
import aiger
import aiger_abc
x = aiger.atom('x')
f = x ^ x
f2 = aiger_abc.simplify(f)
print(f2.aig)
```

and got following errors:

Traceback (most recent call last):
  File "test.py", line 39, in <module>
    f2 = aiger_abc.simplify(f)
  File "/home/dachshund/.local/lib/python3.8/site-packages/aiger_abc/simplify.py", line 29, in simplify
    return aiger.parser.load(aag_path)
  File "/home/dachshund/.local/lib/python3.8/site-packages/aiger/parser.py", line 287, in load
    return parse(''.join(f.readlines()), to_aig=to_aig)
  File "/home/dachshund/.local/lib/python3.8/site-packages/aiger/parser.py", line 278, in parse
    node_map={n: lit2expr[lit] for n, lit in outputs.items()},
  File "/home/dachshund/.local/lib/python3.8/site-packages/aiger/parser.py", line 278, in <dictcomp>
    node_map={n: lit2expr[lit] for n, lit in outputs.items()},
KeyError: 0

It seems like the parser can't handle with zero values in a .aag file. A simplest example given in aiger format document leads to the same problem:

aag 0 0 0 1 0
0

I wonder if it is caused by incorrect py-aiger version, or other reasons I haven't noticed. Looking forward to your reply.

**Comment by mvcisback** [comment]
@Acdimy what version of py-aiger are you using?

**Comment by Acdimy** [comment]
@mvcisback 6.1.23

**Comment by Gy-Hu** [comment]
I also encountered this issue

**Comment by Gy-Hu** [comment]
I temporarily solved this issue by downgrade the py-aiger. Use `pip install -Iv py-aiger==6.1.3` please @Acdimy . However, it may lose some new functionality in the newer verision of py-aiger.

**Comment by mvcisback** [comment]
Thanks!

I think this in fact a regression. I will try to find sometime to fix.

**Comment by mvcisback** [comment]
hey, just wanted to update that i haven't forgotten about this.

Should have some free time to look at the in the next few weeks

**Comment by mvcisback** [comment]
Hi, please check the `main` branch. Parsing of degenerate aag's should be fix.

Apologies for the regression. This seemed to have been introduced during the parser re-write.

**Comment by mvcisback** [comment]
Closing for now. Seems to be fixed in `6.1.24` do to 09e316aa7418e1aab0d95b96814cda6d1c281397

Please open if the issue isn't resolved.
